python/docs/source/user_guide/sql/python_data_source.rst

```diff
@@ -311,6 +350,18 @@ Use the fake datasource with a different number of rows:
     # | Douglas James|2007-01-18|  46226|     Alabama|
     # +--------------+----------+-------+------------+
 
+Write to the fake datasource:
+
+To write data to a custom sink, make sure that you specify the `mode()` clause. Supported modes are `append` and `overwrite`.
```

Comment:
```suggestion
To write data to a custom location, make sure that you specify the `mode()` clause. Supported modes are `append` and `overwrite`.
```

Sink is actually use as a streaming term so I would avoid using it.

Reply:
Sounds good!